src/shared/components/lazyMobXTable/LazyMobXTable.tsx

```diff
@@ -208,29 +213,33 @@ class LazyMobXTableStore<T> {
     @computed public get downloadData()
```

Comment:
I think use case depends on user. I would reckon that the folks who want an excel style output format, might want to get the filtered state, so they don't have to do any post processing. Bioinformaticians are probably happy with a raw data dump.